tests/test_categorical_encoder.py

```diff
@@ -446,7 +446,7 @@ def test_OneHotCategoricalEncoder(dataframe_enc_big, dataframe_enc_big_na):
         encoder.transform(dataframe_enc_big_na)
 
 
-def test_RareLabelEncoder(dataframe_enc_big, dataframe_enc_big_na):
+def test_RareLabelEncoder(dataframe_enc_big, dataframe_enc_big_na, dataframe_enc_rare):
     # test case 1: defo params, automatically select variables
```

Comment:
could you make this file a separate PR?

Reply:
I was under the assumption that this will be treated as a separate one. I will drop it and create another.